tardis/io/config_reader.py

```diff
@@ -27,7 +27,6 @@
 
 default_config_definition_file = os.path.join(data_dir,
```

Comment:
No, removed all the references to `tardis_config_definition.yml` and the file itself.